src/ui/sourceControl/components/stagedFileComponent.svelte

```diff
@@ -11,7 +11,6 @@
     export let view: GitView;
     export let manager: GitManager;
     let buttons: HTMLElement[] = [];
-    $: formattedPath = change.vault_path;
```

Comment:
I removed this line to suppress the following error.

```
14:8   error  'formattedPath' is defined but never used  @typescript-eslint/no-unused-vars
```

This variable is not actually referenced and should not affect the behavior.